--- pcsc-tools-1.4.16/pcsc_scan.c.orig	2009-05-14 20:39:23.000000000 +0200
+++ pcsc-tools-1.4.16/pcsc_scan.c	2010-08-08 19:59:44.678816591 +0200
@@ -69,8 +69,8 @@
 	int current_reader;
 	LONG rv;
 	SCARDCONTEXT hContext;
-	SCARD_READERSTATE_A *rgReaderStates_t = NULL;
-	SCARD_READERSTATE_A rgReaderStates[1];
+	SCARD_READERSTATE *rgReaderStates_t = NULL;
+	SCARD_READERSTATE rgReaderStates[1];
 	DWORD dwReaders = 0, dwReadersOld;
 	DWORD timeout;
 	LPSTR mszReaders = NULL;
